Makefile

```diff
@@ -244,17 +244,17 @@ test-server-race: test-te-race test-ee-race ## Checks for race conditions.
 do-cover-file: ## Creates the test coverage report file.
 	@echo "mode: count" > cover.out
 
-go-junit-report:
+go-junit-report: ## go get -u github.com/jstemmer/go-junit-report
```

Comment:
I'm not sure if this needs to be documented, because is just part of the `test-server` task. Besides, if we want to document this, I would use something like `## Installs the go-junit-report tool`

Reply:
If it doesn't need it then yeah I'd rather not document it. I wasn't sure where it was used.